required/tools/longtable.dtx

```diff
@@ -1223,7 +1225,7 @@
 %    \begin{macrocode}
 \def\LT@start{%
   \let\LT@start\endgraf
-  \endgraf\penalty\z@\vskip\LTpre
+  \endgraf\penalty\z@\vskip\LTpre\endgraf
```

Comment:
I think the additional \endgraf is weird enough to deserve an explanation, after all we are in vertical mode at this point. This is to get the page builder executed I presume? If so th should be mentioned.

Reply:
@FrankMittelbach  The problem is that in the next line \dimen@ is set to \pagetotal but without the \endgraf this doesn't take the \LTpre into account as \pagetotal is updated only after the \endgraf and so the calculation is wrong.

Reply:
I understand (this is triggering the page builder as I assumed) but I think it should be mentioned/documented that it is there for this reason